app/views/posts/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1>The specific post</h1>
\ No newline at end of file
+<div class="fullPost">
+  <div class="postHead">
```

Comment:
### Note: Acknowledge
- [ ] I am able to navigate to user show page manually by entering the id's manually, like for example, as show in the screenshot below

![Screenshot from 2022-04-09 09-00-32](https://user-images.githubusercontent.com/46388113/162564722-bf32a293-0935-4ac7-a333-d28d2d9d7942.png)

Reply:
Yes. I created the views individually. I was not aware that we had to add links as yet.